### PR TITLE
Merging Develop to Master

### PR DESCRIPTION
**Pull request by as14680**
Updating master and prod system with sprints update!!

### CODE REVIEW

**Review by guptaviha — APPROVED** [review]
lgtm

File: backend/NycBasics/models.py [hunk 1]
```diff
@@ -32,3 +32,26 @@ class parking_model(models.Model):
...
     bench_latitude = models.FloatField(null=True)
     bench_longitude = models.FloatField(null=True)
+
+
+class Rating_Review(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    amenity_type = models.CharField(max_length=10, null=False)
+    amenity_id = models.IntegerField(null=False, default=0)
+    # water_id = models.ForeignKey(water_model, on_delete=models.CASCADE)
+    # bench_id = models.ForeignKey(bench_model, on_delete=models.CASCADE)
```
Comment (ds6332) [inline backend/NycBasics/models.py:42]: why are these lines commented?

**Review by ds6332 — COMMENTED** [review]
lgtm

File: backend/NycBasics/models.py (hunk 1 above)
Comment (TheHarbinger99) [reply backend/NycBasics/models.py:42]: trying for a different approach , but the current code has good security and functionality.

**Review by TheHarbinger99 — APPROVED** [review]
lgtm

**Pull request merged**